collect_app/src/test/java/org/odk/collect/android/utilities/ArrayUtilsTest.java

```diff
@@ -0,0 +1,41 @@
+package org.odk.collect.android.utilities;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+
+public class ArrayUtilsTest {
+    @Test
+    public void toPrimitive() throws Exception {
+        Long[] object = new Long[] {1L, 2L, 3L};
+        long[] primitive = new long[] {1, 2, 3};
+        assertArrayEquals("toPrimitiveTest", primitive, ArrayUtils.toPrimitive(object));
```

Comment:
I suggest you remove the first argument, since it adds no value. If the test fails, the name of the test method will appear already.